packages/contracts-bedrock/test/L2/GasPriceOracle.t.sol

```diff
@@ -351,4 +353,46 @@ contract GasPriceOracleFjordActive_Test is GasPriceOracle_Test {
         uint256 upperBound = gasPriceOracle.getL1FeeUpperBound(data.length);
         assertEq(upperBound, 111214);
     }
+
+    /// @dev Tests that `operatorFee` is 0 is Isthmus is not activated.
+    function test_getOperatorFee_succeeds() external view {
+        assertEq(gasPriceOracle.getOperatorFee(10), 0);
```

Comment:
could you assert that is isthums is false? unclear from just looking at this test, could be refactored later and break silently

Reply:
Done